slackbot/dispatcher.py

```diff
@@ -98,7 +98,7 @@ def _default_reply(self, msg):
         default_reply = [
             u'Bad command "%s", You can ask me one of the following questions:\n' % msg['text'],
         ]
-        default_reply += [u'    • `%s`' % str(p.pattern) for p in self._plugins.commands['respond_to'].iterkeys()]
+        default_reply += [u'    • `%s`' % unicode(p.pattern) for p in self._plugins.commands['respond_to'].iterkeys()]
```

Comment:
makes sense but in my opinion it is better to use `u'    • `{}`.format(p.pattern)` and rely on unicode used everywhere.